src/pages/back-office/sing-in/index.js

```diff
@@ -86,7 +87,9 @@ const SingIn = () => {
                 type="string"
                 label={translations["email"]}
                 value={formik.values["email"]}
-                onChange={(e) => formik.setFieldValue("email", e)}
+                onChange={useCallback((e) => formik.setFieldValue("email", e), [
```

Comment:
Seriously? Why U not create a function? It looks unusually

Reply:
added handleChange function